### PR TITLE
Feature/error pages

### PR DESCRIPTION
**Pull request by andresgnlez**

### CODE REVIEW

**Comment by tiagojsag** [comment]
I dont understand why we have the images duplicated. can you explain, please?

**Comment by tiagojsag** [comment]
Missing attribution

**Comment by tiagojsag** [comment]
- [ ] Missing attributions
- [ ] Header style is different (maybe ask dani if we can live without the header menu on both pages, keeping just the logo?)
- [ ] Logo has different sizes

File: app/src/components/Shared/NoFoundPage.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import Header from '../../containers/Header';
+import StatusPage from './StatusPage';
+
+class NotFoundPage extends Component {
```
Comment (clementprdhomme) [inline app/src/components/Shared/NoFoundPage.jsx]: The file is called `NoFoundPage` and and the component `NotFoundPage`. 😉

**Comment by clementprdhomme** [comment]
I don't think we need them to be duplicated. In the Javascript folder, we don't need the image for the page 500, and in `public` we don't need the one for 404.

**Comment by tiagojsag** [comment]
thought so too, but I didnt want to assume too much, being the front noob around here ;)

**Comment by tiagojsag** [comment]
Attribution on the 500 page has incorrect font

**Comment by tiagojsag** [comment]
- [ ] On 500, the mobile header and mobile menu are different than on the rest of the pages
- [ ] The 500 page has login selected
- [ ] On mobile menu, make the 'p' in "Research program" upper case

**Pull request merged**